docs/concepts/cluster-administration/device-plugins.md

```diff
@@ -45,12 +45,30 @@ For example, after a device plugin registers `vendor-domain/foo` with the kubele
 and reports two healthy devices on a node, the node status is updated
 to advertise 2 `vendor-domain/foo`.
 
-Then, developers can request devices in a
+Then, users can request devices in a
 [Container](/docs/api-reference/{{page.version}}/#container-v1-core)
-specification by using the same process that is used for
-[opaque integer resources](/docs/tasks/configure-pod-container/opaque-integer-resource/).
-In version 1.8, extended resources are supported only as integer resources and must have
-`limit` equal to `request` in the Container specification.
+specification as they request other types of resources, with the following limitations:
+  * Extended resources are only supported as integer resources, and must have `limit` equal to `request` in the Container specification.
```

Comment:
Unless there are other constraints on device resources beyond what extended resources require, limit must equal request only if both are specified. For vanilla extended resource at least, it is valid to omit limit.

Reply:
Hmm, I checked the ValidateResourceRequirements() code and looks like the current implementation is like what you described. But that feels like a bug. Allowing users to just set request for extended resources may mislead them to feel we support burstable consumption on such resources, but Kubelet currently has no way to support this. I think that is the reason that we explicitly disallow the case of only setting request for gpus: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go#L4375

Reply:
Unfortunately I think that ship has sailed already as those are the v1.8 semantics. Changing that at this point would break user pod specs.

Reply:
We have documented that ERs do not support overcommit, FWIW.

Reply:
That is true. The ER doc does clearly say that extended resources cannot be overcommitted. I modified this sentence to be consistent with the ER documentation and also use the ER doc link above instead of the PR link. In that sense, it perhaps less a concern that the API validation doesn't throw an error for such resource spec. I will include some scheduling folks on my validation change PR to get their opinions.